### PR TITLE
LPS 148593 

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang ,

This pull is to remove unnecessary PowerMock in portal-impl and portal-kernel. All Test case run successfully in my local.

Main Task: LPS-84905 Remove Mockito and PowerMockito
Sub Task: LPS-148593 Remove unnecessary PowerMock in portal-impl and portal-kernel

Thanks for Checking!
Janis

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

File: portal-impl/test/unit/com/liferay/portal/servlet/ComboServletTest.java [hunk 1]
```diff
@@ -194,8 +187,6 @@ public void testGetResourceRequestDispatcherWithoutPortletId()
 			_mockHttpServletRequest, _mockHttpServletResponse,
 			"/js/javascript.js");
 
-		Mockito.verify(_portalServletContext);
```
Comment (dantewang) [inline portal-impl/test/unit/com/liferay/portal/servlet/ComboServletTest.java:197]: @JanisZhang Here the original code uses verify to make sure the `_portalServletContext` has been called once. If you remove the mock usage, I think you need to find another way to assert it? e.g. a proxy with a counter in its `InvocationHandler`.

**Comment by dantewang** [comment]
@JanisZhang It seems the replacements of mocking certain method behavior (return value, etc) are correct.

However, like I commented above, if assertions are done by the mock framework (like Mockito.verify?), they are lost after your replacement. Please double check these changes, thanks!

**Comment by JanisZhang** [comment]
Ok

**Comment by JanisZhang** [comment]
Hi @dantewang,
I have replaced the 2 of 3 `Mockito.verify()` usages by using CountingInvocationHandler. However, it looks hard to replace `inOrder.verify` in FallbackSettingsTest, so I keep it.  And I also add a new commit to replace power mock usage. 
Thanks for checking

**Comment by JanisZhang** [comment]
ci:test:sf

**Pull request closed without merging**